### PR TITLE
[Intl] Update the ICU data to 70.1 - 4.4

### PR DESCRIPTION
**Pull request by jderusse**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/Intl/Intl.php [hunk 1]
```diff
@@ -256,7 +256,7 @@ public static function getIcuDataVersion(): string
      */
     public static function getIcuStubVersion(): string
     {
-        return '68.2';
```
Comment (jderusse) [inline src/Symfony/Component/Intl/Intl.php:259]: Looks like we skiped 69.1

**Comment by fabpot** [comment]
Tests do not run apparently for our intl data tests.

**Comment by jderusse** [comment]
tests green :green_heart:

**Comment by fabpot** [comment]
Can you revert the whitespace changes in the yml file as I don't think they are needed.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @jderusse.

**Pull request merged**